python-mini-projects/projects/Shutdown_or_restart_your_device/PowerOptions.py

```diff
@@ -0,0 +1,28 @@
+import os
+import platform
+
+def shutdown():
+    if platform.system() == "Windows":
+        os.system('shutdown -s')
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This line of code makes an outdated API call to start and communicate with processes. We recommend that you use the `subprocess` module to start new processes, connect with their pipes, and get their return codes.

[Learn more](https://docs.python.org/3/library/subprocess.html)

Similar issue at line number 8.